server/src/facepp.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const https = require('https');
+const qs = require('querystring');
+
+const createForm = obj => qs.stringify(obj);
+
+const fetch = (method, options) => {
+  options = Object.assign(
+    {
+      api_key: config.api.facepp.apiKey,
+      api_secret: config.api.facepp.secret,
+    },
+    options
+  );
+
+  const url = config.resources.facepp + method;
+  const formData = createForm(options);
+  const req = https.request(url, { method: 'POST' });
+  req.setHeader('Content-Type', 'application/x-www-form-urlencoded');
+  req.setHeader('Content-Length', formData.length);
+  req.write(formData);
+  req.end();
+
+  return new Promise((resolve, reject) => {
+    req.on('response', res => {
```

Comment:
I think it would be easier for you to require `node-fetch` to avoid manual buffer handling. Also, you could then keep async/await style trough HTTP requests.